testkit/mocksessionmanager.go

```diff
@@ -26,6 +29,9 @@ type MockSessionManager struct {
 	PS      []*util.ProcessInfo
 	SerID   uint64
 	TxnInfo []*txninfo.TxnInfo
+	Conn    map[uint64]session.Session
+	Dom     *domain.Domain
+	Mu      sync.Mutex
```

Comment:
It better to make `Conn` and `Mu` private to avoid others using `Conn` directly that may cause a data race